addons/mail/static/src/messaging/component/partner_mention_suggestion/partner_mention_suggestion.js

```diff
@@ -0,0 +1,76 @@
+odoo.define('mail.messaging.component.PartnerMentionSuggestion', function (require) {
+'use strict';
+
+const useStore = require('mail.messaging.component_hook.useStore');
+
+const components = {
+    PartnerImStatusIcon: require('mail.messaging.component.PartnerImStatusIcon'),
+};
+
+const { Component } = owl;
+
+class PartnerMentionSuggestion extends Component {
```

Comment:
This component should not rely on composer at all, but only partner and maybe a prop for whether it should be highlighted or not.

Reply:
Done